src/Screens/Onboarding/Email.tsx

```diff
@@ -47,56 +47,79 @@ const Email = ({ navigation }: Props) => {
   return (
     <OnboardingBackground testID="onboarding.email.view">
       <Card style={styles.card}>
-        <Message style={styles.title} id="onboarding.email.title" />
-        <NamedInputField
-          autoCapitalize="none"
-          style={styles.nickNameInput}
-          name="onboarding.email.inputTitle"
-          onChangeText={setEmail}
-          autoCompleteType="off"
-          value={email}
-          testID="onboarding.email.inputTitle"
-        />
-        <Message style={styles.bodyText} id="onboarding.email.bodyText" />
-        <Button
-          style={styles.nextButton}
-          onPress={navigateNext}
-          messageId="onboarding.email.nextButton"
-          badge={require('../images/arrow.svg')}
-          testID="onboarding.email.nextButton"
-          disabled={!isValidEmail}
-        />
-        <Button
-          messageId="onboarding.signUp.back"
-          onPress={goBack}
-          noShadow={true}
-          style={{ backgroundColor: colors.lightestGray }}
-        />
+        <RN.View>
```

Comment:
This View looks kinda excess?

Reply:
its needed , because its a flex container